### PR TITLE
Add conda environment to some of our tests

### PR DESCRIPTION
**Pull request by rchiodo**
For #4162

Add conda environments to functional tests.

-   [ ] Pull request represents a single change (i.e. not fixing disparate/unrelated things in a single PR).
-   [ ] Title summarizes what is changing.
-   [ ] Has a [news entry](https://github.com/Microsoft/vscode-jupyter/tree/main/news) file (remember to thank yourself!).
-   [ ] Appropriate comments and documentation strings in the code.
-   [ ] Has sufficient logging.
-   [ ] Has telemetry for enhancements.
-   [ ] Unit tests & system/integration tests are added/updated.
-   [ ] [Test plan](https://github.com/Microsoft/vscode-jupyter/blob/main/.github/test_plan.md) is updated as appropriate.
-   [ ] [`package-lock.json`](https://github.com/Microsoft/vscode-jupyter/blob/main/package-lock.json) has been regenerated by running `npm install` (if dependencies have changed).

### CODE REVIEW

**Comment by rchiodo** [comment]
Actually closing for now. I didn't reenable conda for the VS code tests.

**Comment by rchiodo** [comment]
I believe I have the VS code tests working now too.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -527,6 +580,18 @@ jobs:
           python -m pip install --upgrade -r ./build/conda-functional-requirements.txt
           python -m ipykernel install --user
 
+      - name: Install Python Libs for conda
+        shell: bash -l {0}
+        if: matrix.test-suite == 'conda'
+        run: |
+          conda activate functional_test_env
```
Comment (IanMatthewHuff) [inline .github/workflows/main.yml:587]: Doesn't seem like a bit deal either way, but just out of interest. Looks like that Conda github action is already running an activate-environment step, do we need this activate here?

**Review by IanMatthewHuff — APPROVED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (rchiodo) [reply .github/workflows/main.yml:587]: Yes. In github actions each step is a different bash shell.

**Review by DonJayamanne — APPROVED** [review]

**Pull request merged**